src/app/public/modules/split-view/split-view-drawer.component.ts

```diff
@@ -0,0 +1,207 @@
+import {
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  HostListener,
+  Input,
+  OnDestroy,
+  OnInit
+} from '@angular/core';
+
+import {
+  SkyCoreAdapterService,
+  SkyMediaBreakpoints,
+  SkyMediaQueryService
+} from '@skyux/core';
+
+import {
+  Observable,
+  Subject
+} from 'rxjs';
+
+import {
+  SkySplitViewMediaQueryService
+} from './split-view-media-query.service';
+
+import {
+  SkySplitViewService
+} from './split-view.service';
+
+let nextId = 0;
```

Comment:
Since this will be global could we make it a little more specific?

Reply:
Done.